src/components/list/InfiniteAutoRowHeightList.js

```diff
@@ -21,50 +21,70 @@ const InfiniteAutoRowHeightList = memo(
-              mode="cells"
-              columnCount={1}
-              rowCount={list.length}
-            >
-              {({ onSectionRendered, scrollToRow }) => (
-                <List
-                  ref={registerChild}
-                  width={width}
-                  height={height}
-                  {...(isKeyboardNavigationEnabled && {
-                    scrollToIndex: scrollToRow,
-                  })}
-                  {...(isKeyboardNavigationEnabled && {
-                    onSectionRendered,
-                  })}
-                  rowHeight={({ index }) => rowHeight(list, index)}
-                  rowCount={list.length}
-                  overscanRowCount={overscanRowCount}
-                  onRowsRendered={onRowsRendered}
-                  rowRenderer={({ index, key, style }) =>
-                    rowRenderer({ index, key, style, scrollToRow })
-                  }
-                  noRowsRenderer={noRowsRenderer}
-                />
-              )}
-            </ArrowKeyStepper>
-          )}
-        </AutoSizer>
-      )}
-    </InfiniteLoader>
-  ),
+  }) => {
+    const keyStepperRef = useRef()
+
+    return (
+      <InfiniteLoader
+        isRowLoaded={({ index }) => isRowLoaded(index)}
+        loadMoreRows={loadMoreRows}
+        rowCount={list.length + threshold}
+        minimumBatchSize={minimumBatchSize}
+        threshold={threshold}
+      >
+        {({ onRowsRendered, registerChild }) => (
+          <AutoSizer disableWidth>
+            {({ height }) => (
+              <ArrowKeyStepper
+                ref={keyStepperRef}
+                disabled={!isKeyboardNavigationEnabled}
+                mode="cells"
+                columnCount={1}
+                rowCount={list.length}
+              >
+                {({ onSectionRendered, scrollToRow }) => (
+                  <List
+                    ref={registerChild}
+                    width={width}
+                    height={height}
+                    {...(isKeyboardNavigationEnabled && {
+                      scrollToIndex: scrollToRow,
+                    })}
+                    {...(isKeyboardNavigationEnabled && {
+                      onSectionRendered,
+                    })}
+                    rowHeight={({ index }) => rowHeight(list, index)}
+                    rowCount={list.length}
+                    overscanRowCount={overscanRowCount}
+                    onRowsRendered={onRowsRendered}
+                    rowRenderer={({
+                      index,
+                      key,
+                      isVisible,
+                      isScrolling,
+                      style,
+                    }) => {
+                      if (
+                        isKeyboardNavigationEnabled &&
+                        isVisible &&
+                        !isScrolling
+                      ) {
+                        keyStepperRef.current.setScrollIndexes({
+                          scrollToRow: index,
```

Comment:
now it doesn't jump to the first user but still jumps.

as soon as is top scrolling, I think it scrolls the the last visible user currently, which is almost never exactly at the scroll position where the code wants it to be, so this looks like a confusing jump.